ui/src/alerting/selectors/index.ts

```diff
@@ -14,3 +15,17 @@ export const getResourceIDs = (
     {}
   )
 }
+
+interface HasName {
+  name?: string
+}
+
+export const getResourceList = <T extends HasName>(
+  state: AppState,
+  resource: ResourceType
+): T[] => {
+  const resourceList: T[] = get(state, `${resource}.list`, [])
+  return sortBy(resourceList, l => {
```

Comment:
Why `_.sortBy` over `Array.prototype.sort`?

Reply:
nice and clean.